mantid-developer-linux.yml

```diff
@@ -39,7 +39,6 @@ dependencies:
   - requests>=2.25.1
   - scipy>=1.6.2
   - setuptools=48.0.0 # Pinned purposefully due to incompatibility with later versions
-  - sphinx<=3.5.4 # Pinned at 3.5.4 and below due to styling changes in 4.0, and 4.3 has a bug fix which imports a lot of matplotlib code (probably correctly, based on what we have in the api/python/mantid/plots/index.html file) and fails the doctests.
```

Comment:
After fixing the mslice update problem on the builds I reran the conda linux build here and it says it can't find Sphinx.

I think we have accidentally removed the Sphinx requirement completely instead of the pin? I think we want
```
- sphinx >= 4.5.*
```

to get any patch version from the 4.5 series?